tools/property-extractor/property_extractor.py

```
@@ -0,0 +1,230 @@
+    transformers = [
+        TypeTransformer(),
+        EnterpriseTransformer(), ## this must be the first, as it modifies current data
+        MetaParamTransformer(),
+        BasicInfoTransformer(),
+        IsNullableTransformer(),
+        IsArrayTransformer(type_transformer),
+        NeedsRestartTransformer(),
+        VisibilityTransformer(),
+        DeprecatedTransformer(),
+        IsSecretTransformer(),
+        NumericBoundsTransformer(type_transformer),
```

Ruff (0.8.2)

`NumericBoundsTransformer` may be undefined, or defined from star imports

(F405)